packages/playground/html/unicode-path/中文-にほんご-한글-🌕🌖🌗/index.html

```diff
@@ -1 +1 @@
-<h1>unicode-path</h1>
+<h1>Unicode-path</h1>
```

Comment:
This one is the only one that I have doubts about. I think that if we want to case Unicode correctly, we should also remove the `-`, if not we should keep it lowercase